stroeer/page/article/v1/article_extender_service.proto

```diff
@@ -33,11 +36,34 @@ service ArticleExtenderService {
  *
  * @CodeBlockStart protobuf
  */
-
 message GetQuestionsRequest {
   int64 id = 1;
 }
 
 message GetQuestionsResponse {
   repeated string questions = 1;
 }
+
+/**
+ * # `⚙︎ GetAnswer`
+ * ```protobuf
+ * rpc GetAnswer(GetAnswerRequest) returns (GetAnswerResponse) {}
+ * ```
+ *
+ * Returns the answer of a question as a string. Otherwise an `Error`.
+ *
+ * | Field name       | Type                | Description                                       |
+ * |------------------|---------------------|---------------------------------------------------|
+ * | `question`       | `string`            | [required] The question for the returned answers. |
+ *
+ * @CodeBlockStart protobuf
+ */
+message GetAnswerRequest {
+  string question = 1;
+}
+
+message GetAnswerResponse {
+  string answer_text = 1;
```

Comment:
Within this message, on the other hand, we can be generic ... stripping the prefix?

- `AnswerResponse.answer: string`
- `AnswerResponse.sources: list[Article]`